src/ground-truth.rkt

```diff
@@ -66,4 +65,9 @@
       (make-sampler ctx pre fn)))
   (timeline-event! 'sample)
   (match-define (cons table2 results) (batch-prepare-points fn ctx sampler))
-  (cons (combine-tables table table2) results))
+  (representation-repr->bf (context-repr ctx))
+  (define combined-table (combine-tables table table2))
+  (when (> (hash-ref combined-table 'infinite 0.0) 0.2)
```

Comment:
I thought about this more, and I've concluded that actually we need to test `table2`, not `combined-table`. (Complicated, can explain in person.) That stores counts, not percentages, so you'll also need to deal with that.